scripts/containers/production-smoke.sh

```
@@ -0,0 +1,611 @@
+  service_systemctl start kravhantering-ci-hsa.target
+  service_systemctl enable --now kravhantering-single-node.target
+  wait_for_url https://kravhantering.test/api/health
+  wait_for_url https://kravhantering.test/api/ready
+  verify_containment
+  service_systemctl restart kravhantering-app-runtime.service
+  wait_for_url https://kravhantering.test/api/ready
+  as_service "$INSTALL_ROOT/current/bin/kravhantering-quadlet.sh" \
+    install --topology "$TOPOLOGY"
+  service_systemctl daemon-reload
+  service_systemctl stop kravhantering-single-node.target
+  ! service_systemctl is-active --quiet kravhantering-single-node.target
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)